packages/ui-components/src/navigation/Payments/Payments.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FC } from 'react';
+import { Image } from '../../data-display/Image'
+import './index.css'
+import paypal from '../../assets/paypal.jpg'
+import bitcoin from '../../assets/bitcoin.jpg'
+import qrcode from '../../assets/qrcode.jpg'
+import bank from '../../assets/bank.jpg'
+
+type Props = {};
```

Comment:
you can add prop for accept supported payment methods, since every web shop defines this when creating/updating an api key
and then conditionally render buttons